app/views/mentor/knowledges/new.html.haml

```diff
@@ -0,0 +1,14 @@
+= form_for ['mentor', @knowledge] do |f|
+  = f.text_field :title, placeholder: "title"
+  = f.text_area :content, cols: "100", row: "100"
```

Comment:
__[HAML-Lint]__

Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.


<!-- Comment by [SideCI](https://sideci.com) -->